install/slackware/passwordsafe.SlackBuild

```diff
@@ -103,69 +107,75 @@ PKGNAME=${PRGNAM}-${VERSION/-/.}-${ARCH}-${BUILD_ID}${BUILDER_ID}
 umask 022
 
 # Create working directories:
-mkdir -p ${LOGS}
-mkdir -p ${OUTPUT}
-mkdir -p ${PKG}
-mkdir -p ${MAKEDIR}
+mkdir -p "${LOGS}"
+mkdir -p "${OUTPUT}"
+mkdir -p "${PKG_TMP}"
+mkdir -p "${MAKEDIR}"
 # always erase old package's contents
-rm -rf ${PKG}/* ${MAKEDIR}/*
+rm -rf "${PKG}"/* "${MAKEDIR}"/*
 
 ## Package building
 echo Building ...
 cd ${MAKEDIR}
-cmake ${FLAGS} ${GIT_ROOT} 2>&1 | tee $LOGS/cmake-${PRGNAM}.log || exit 1
+cmake ${FLAGS} ${GIT_ROOT} 2>&1 | tee "${LOGS}/cmake-${PRGNAM}.log" || exit 1
 
 echo "- make"
-make ${MAKE_OPTIONS} 2>&1 | tee $LOGS/make-${PRGNAM}.log || exit 1
+make ${MAKE_OPTIONS} 2>&1 | tee "${LOGS}/make-${PRGNAM}.log" || exit 1
 
 echo "- prepare package"
-DESTDIR=${PKG} make install 2>&1 | tee $LOGS/install-${PRGNAM}.log || exit 1
+DESTDIR="${PKG_TMP}" make install 2>&1 | tee "${LOGS}/install-${PRGNAM}.log" || exit 1
 
 # collect files before packaging
 # copy docs
-mkdir -p ${PKG}/${DOCDIR}
-cp ${GIT_ROOT}/README* ${GIT_ROOT}/LICENSE ${GIT_ROOT}/install/copyright ${PKG}/${DOCDIR}
-if [-d "${PKG}${DEFDOCDIR}" ]; then
-    mv ${PKG}${DEFDOCDIR}/* ${PKG}/${DOCDIR} || true
-    rmdir ${PKG}/${DEFDOCDIR}
+mkdir -p "${PKG_TMP}${DOCDIR}"
+cp "${GIT_ROOT}"/README* "${GIT_ROOT}"/LICENSE "${GIT_ROOT}"/install/copyright "${PKG_TMP}${DOCDIR}"
```

Comment:
Missing / in `"${PKG_TMP}${DOCDIR}"`?

Reply:
Same below. Did you intend for `PKG_TMP` to end in a `/`?

Reply:
No, (DEF)DOCDIR, (DEF)MANDIR, ... already have leading slash because prefix is `/usr`